lerna.json

```diff
@@ -0,0 +1,22 @@
+{
+  "version": "0.1.21",
+  "npmClient": "npm",
+  "command": {
+    "publish": {
+      "conventionalCommits": "true",
+      "ignoreChanges": [
+        "ignored-file",
+        "*.md",
+        "*,yaml"
```

Comment:
should be .yaml

maybe also add .yml

Reply:
Good deal. Fixing it. The `,` is the way it is in master, and didn't look too hard at the syntax, though I do realize it is borked.

Reply:
Also added `.yml`